acceptance/gem_versions/.acceptance/acceptance-cookbook/recipes/destroy.rb

```diff
@@ -0,0 +1,3 @@
+execute 'bundle exec kitchen destroy --log-level debug' do
```

Comment:
I would prefer to keep it until we stabilize things a bit to understand what's up if things go weird. Also I'd like to get this merged so that I can close out the jenkins work. We can change or get rid off the suite afterwards if we want to.